docs/best-practices.md

```diff
@@ -202,9 +202,42 @@ An admonition paragraph is rendered in a callout box with the admonition label 
 
 Asciidoctor provides five admonition style labels: `NOTE`, `TIP`, `IMPORTANT`, `CAUTION` and `WARNING`
 
-An `admonition` is written in following example style: `[<label>:]` Text
+### Simple Admonitions
 
-Example: `NOTE:` If you have a large ownCloud installation and have shell access, you should ...
+Use this way when you just have text to write.
+A simple `admonition` is written in the following style: `[<label>:]` Text
```

Comment:
[They're not written that way](https://asciidoctor.org/docs/user-manual/#admonition). There's either the short version `<label>:` or the longer version `[<label>]\n`.